hologram_dashboard/fetcher/BUILD

```diff
@@ -6,19 +6,30 @@ cc_library(
 )
 
 cc_library(
-    name = "hologram_data_source_availabiliity_fetcher",
-    srcs = ["hologram_data_source_availabiliity_fetcher.cc"],
+    name = "hologram_data_source_availability_fetcher",
+    srcs = ["hologram_data_source_availability_fetcher.cc"],
     hdrs = ["hologram_data_source_availability_fetcher.h"],
     deps = ["//fetcher/proto:hologram_availability_cc_proto", 
             "//fetcher/proto:hologram_config_cc_proto", 
             "@com_github_gflags_gflags//:gflags",
+            "@com_google_gtest//:gtest",
             "hologram_data_fetcher_lib"],
 )
 
+cc_test(
+    name = "hologram_data_source_availability_fetcher_test",
+    srcs = ["hologram_data_source_availability_fetcher_test.cc"],
+    deps = [
+        "hologram_data_source_availability_fetcher",
+    ],
+    data = ["testdata/hologram_config_test.config", 
+            "testdata/hologram_config_malformed.config"]
+)
+
 cc_binary(
     name = "fetcher_main",
     srcs = ["fetcher_main.cc"],
     deps = [
-        "hologram_data_source_availabiliity_fetcher"
+        "hologram_data_source_availability_fetcher"
```

Comment:
super nit: I feel this file name is too long, and no need to include hologram. Can we name it like data_availability_fetcher?